### PR TITLE
feat: use a keyword argument to retrieve all workitem attributes

### LINKED ISSUE #179: why long attributes are skipped ?
Hi,
I was trying to get as many attributes from work item as I can, and I was stucked [here](https://github.com/dixudx/rtcclient/blob/266768d35808af5cc4fdaa7c8855842e3e227331/rtcclient/base.py#L310-L313), is there a reason to have this check to skip "long attributes" ?

I've removed that block of code, on 0.6.0 version, and finally I can get all the workitem's attributes (some of them are still links to be parsed).

Let me know, thanks.

### PR DESCRIPTION
**Pull request by gpongelli**
added kwargs to getWorkItem to manage user's need

Fixes #179

### CODE REVIEW

File: rtcclient/base.py [hunk 1]
```diff
@@ -17,12 +17,17 @@ class RTCBase(object):
     OSLC_CR_XML = "application/x-oslc-cm-change-request+xml"
     OSLC_CR_JSON = "application/x-oslc-cm-change-request+json"
 
-    def __init__(self, url):
+    def __init__(self, url, **kwargs):
```
Comment (dixudx) [inline rtcclient/base.py:20]: Shall we use `__init__(self, url, skip_full_attributes=True, **kwargs)` instead

**Review by dixudx — COMMENTED** [review]
Nice work!

File: rtcclient/base.py (hunk 1 above)
Comment (gpongelli) [reply rtcclient/base.py]: Done

**Comment by gpongelli** [comment]
I see merge conflict, I’ll resolve them after a rebase.

**Pull request merged**